policy/approval/approve.go

```diff
@@ -258,7 +259,14 @@ func (r *Rule) filteredCandidates(ctx context.Context, prctx pull.Context) ([]*c
 	sort.Stable(common.CandidatesByCreationTime(candidates))
 
 	if r.Options.IgnoreEditedComments {
-		candidates, err = r.filterEditedCandidates(ctx, prctx, candidates)
+		candidates, err = r.filterEditedCandidates(ctx, prctx, candidates, "comments")
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if r.Options.IgnoreEditedBody {
```

Comment:
Looking at this again, the way the "candidates" abstraction currently works means that we can only have a single option for ignoring edited candidates. Because `filterEditedCandidates` can't tell if a candidate came from a comment or a body, the first call to `filterEditedCandidates` will remove all edits, regardless of which flag is set.

If you think a single flag will work for your use case, let's remove the `ignore_edited_body` flag and update the docs for `ignore_edited_comments` to clarify that it applies to bodies too. If you think separate flags are required, I think we'll have to introduce a `Type` field  and a `CandidateType` "enum" to the `Candidate` struct so that the filtering can distinguish between types of candidates if needed.